src/index.js

```diff
@@ -24,6 +25,10 @@ class Dropzone extends React.Component {
 
   componentDidMount() {
     this.enterCounter = 0;
+    const { handleOnCancel } = this;
+    document.body.onfocus = (event) => {
```

Comment:
You should properly attach the event listener to the DOM node using `addEventListener` method. Also, you forgot to remove the listener in `componentWillUnmount`